lib/src/bbox.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:turf/helpers.dart';
+import 'package:turf/meta.dart';
+
+BBox bbox(GeoJSONObject geoJson, {Map<String, dynamic>? options}) {
```

Comment:
I think we could simplify this a little by creating a parameter `{bool recompute = false}`, we could entirely get rid of `options` then.

It would also simplify the following if clause to something like this:
```dart
if (geoJson.bbox != null && !recompute) {
```